helm/prometheus-rules/templates/recording-rules/service-level.rules.yml

```diff
@@ -238,7 +238,7 @@ spec:
         class: MEDIUM
       record: raw_slo_errors
       # -- 99% availability
-    - expr: "vector((1 - 0.99))"
+    - expr: label_replace(group(rest_client_requests_total{app=~"kube-controller-manager|kube-scheduler"}) by (cluster_id, app), "service", "$1", "app", "(.*)") * 0 + 1 - 0.99
```

Comment:
Keeping 99% availability but adding the missing labels

![image](https://user-images.githubusercontent.com/6536819/151942378-60f9a34b-867b-4771-b45c-57c0b23ff7a4.png)